wordpress-mobile/WordPress-iOS
# Remove stats from Core Data

Related #20773

More details in https://github.com/wordpress-mobile/WordPress-iOS/pull/21067

This PR removes stats from Core Data. The cache is not stored outside of Core Data

## To Test

- Install the previous production release version
- Browse the Stats (both the Insights and the stats be periods)
- Upgrade to the version form this PR
- Verify that the migration is performed and the app continues to work

I used a DB browser to confirm that the tables were removed. There were only two: `ZSTATSRECORD` and `ZSTATSRECORDVALUE` (one table containing all columns for all the stats record subclasses).

## Regression Notes
1. Potential unintended areas of impact: Core Data store
2. What I did to test those areas of impact (or what existing automated tests I relied on): manual test
3. What automated tests I added (or what prevented me from doing so) n/a

PR submission checklist:


UI Changes testing checklist:
